### ISSUE TITLE
Support for variable table name

### ISSUE THREAD

**Issue by weizmann** [Raizlabs/DBFlow#697]
It is quite amazing when integrating my app with DBFlow~

In my app, different user has different tables to store their own private data. I think put different user data into a single table is not pleased. So I put different user data into different tables.

In my previous implement, I just created table in format: **$tablename_$userid**, e.g. call-log table for John(userid:1122) is calllog_1122, and call-log table for Mike(userid:3344) is calllog_3344.
In short, table name is a variable. But it seemed impossible when using DBFlow, as the table-name annotation field requires a constant variable;

Could you please help me find some solutions about this problem? 
Thank you very much~

**Comment by VankaIn** [comment]
[`#687`](https://github.com/Raizlabs/DBFlow/issues/689)

**Comment by agrosner** [comment]
This is not supported in the lib, since it depends on code gen, making it nearly impossible to support.
